doc/nrf/ug_nrf5340.rst

```diff
@@ -0,0 +1,155 @@
+
+These samples must be programmed to the application core, in the secure domain.
+
+
+Building and programming a sample
+*********************************
+
+You must program both the network sample and one of the supported :ref:`Bluetooth Low Energy samples <ble_samples>`.
+
+Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
+Make sure to use ``nrf5340_dk_nrf5340_cpunet`` as board name when building the network sample, and ``nrf5340_dk_nrf5340_cpuapp`` when building the application sample.
+
+.. important::
+   When programming the samples from |SES|, you might get an error message stating that the target cannot be identified.
+   In this case, you can either modify your |SES| installation and projects to add support for programming nRF5340, or program the generated HEX files from the command line instead.
+
+   See the following sections for more information.
+
+
+Adding support for programming nRF5340 in |SES|
+===============================================
+
+The current version of the Nordic Edition of |SES| (v4.20a) does not include the latest SEGGER J-Link version (v6.54c).
+However, the latest version is required to program nRF5340 devices.
+
+To add support for programming nRF5340 in |SES|, complete the following steps:
+
+1. Download and install the latest `J-Link Software and Documentation Pack`_ (v6.54c or later).
+#. Copy the :file:`JLink_x64.dll` file from the J-Link installation directory into the ``bin`` folder of your |SES| (Nordic Edition) folder, replacing the existing file.
+   On Windows, this file is by default located in ``C:\Program Files (x86)\SEGGER\JLink\JLink_x64.dll``.
+   Note that the file might be named differently for other operating systems.
+#. Restart |SES|.
+#. Open an |NCS| project with the full path to ``boards/arm/nrf5340_dk_nrf5340`` in the Zephyr repository as board directory and either ``nrf5340_dk_nrf5340_cpunet`` or ``nrf5340_dk_nrf5340_cpuapp`` as board name.
+#. Right-click on the project in the Project Explorer and select **Options**.
+#. Navigate to **Debug** > **Debugger**.
+#. Double-click the value for **Target Device** and select **nRF9160** from the list.
+   nRF5340 is not included in the list yet, but selecting nRF9160 sets the required configuration.
+#. Double-click the value for **Target Device** again and enter **nRF5340** in the search field.
+   Click **OK** to use this value.
+
+You can now build and program the sample for nRF5340.
+
+
+Programming from the command line
+=================================
+
+To program a HEX file after building it with |SES|, open a command prompt in the build folder of the sample that you want to program and enter the following command::
+
+    west flash
+
+If you prefer to use nrfjprog (which is part of the `nRF Command Line Tools`_) instead, open a command prompt in the build folder of the network sample and enter the following commands to program the network sample::
```

Comment:
Is the folder for building the same as in line 141? If not, I would add italics around the first `network` to make it stand out that it's a different folder. If yes, I would rephrase for consistency.

Reply:
Line 141 applies to building either sample.
The instructions for nrfjprog differ depending on which sample you build.

Reply:
I see. I think this part would benefit from re-writing anyway, as the two paragraphs look like alternatives, because of this "If... instead".